packages/react-native/src/ReactNativeFileSystem.ts

```diff
@@ -21,6 +21,10 @@ export class ReactNativeFileSystem implements FileSystem {
     return RNFS.exists(path)
   }
 
+  public async createDirectory(path: string): Promise<void> {
```

Comment:
Annoying we need to create the directory ourselves for Askar. I would like to use the file system as little as possible, oh wel...